script/build

```diff
@@ -14,6 +14,7 @@ find_source_files() {
 build_hub() {
   mkdir -p "$(dirname "$1")"
   go build \
+      "$MOD_VENDOR_ARG" \
```

Comment:
This should not have quotes. If MOD_VENDOR_ARG is empty, you do not want an empty string argument to be passed. You want no argument to be passed.

Reply:
It also used space indents instead of tabs, that is fixed too.